server/graphql/v1/queries.js

```diff
@@ -295,6 +296,7 @@ const queries = {
         startDate: args.dateFrom,
         endDate: args.dateTo,
         includeExpenseTransactions: args.includeExpenseTransactions,
+        kinds: args.kinds,
```

Comment:
If it was API v2, I think I would insist on making it singular `kind` (still allowing a list).

I'm more relax on GraphQL v1 and change is not required.

Reply:
The same `kinds` field actually exists on GQLV2: https://github.com/opencollective/opencollective-api/blob/main/server/graphql/v2/query/TransactionsQuery.ts#L81. Having a plural here makes sense to me since we only accept a list. It's similar to what we do with the `tags` argument for example.

Reply:
I think the DX is easier with singular, no need to think if it should be singular or plural, just use the name of field. That way, you can also accept a single element without feeling weird. In GraphQL, when the argument is a list, you can pass a single item, which is nice. In this case that would be`kinds: `PLATFORM_TIP`. With singular, it's also closer to its Sequelize / SQL equivalent.

This was a conscious design in the early days of GraphQL v2:

https://github.com/opencollective/opencollective-api/blob/main/server/graphql/v2/interface/Account.js#L155
https://github.com/opencollective/opencollective-api/blob/main/server/graphql/v2/interface/Account.js#L204

I haven't been vigilant enough and I should have catch `types` and `tags` when they were introduced.